hack/build-go.sh

```diff
@@ -55,7 +55,7 @@ if [ $# -eq 0 ]; then
             go $target -tags selinux ./...
 
             cd ${KUBEVIRT_DIR}/staging/src/kubevirt.io
-            go $target ./...
+            GO111MODULE=off go $target ./...
```

Comment:
Maybe we should do this globally in the dockerized container?

Reply:
That is not necessary in my opinion. Note that this can be also fixed like this:
`cd ${KUBEVIRT_DIR}/staging/src/kubevirt.io/client-go
GOFLAGS= go $target ./...
 `
(Suppressing -mod=vendor as we don't have a vendor there and its outside main module)

Reply:
I think that it's better to turn off modules globally because we may encounter this in other places.
I don't see why we should turn it off every time instead of turning it off globally for all of our tools.